test/e2e/todoFeatures.js

```diff
@@ -0,0 +1,60 @@
+describe('To Do List', function() {
+  beforeEach(function(){
+    browser.get('http://localhost:8080');
+  })
+
+  it('allows list items to be displayed', function(){
+    var addItem = element(by.model('todoCtrl.newItem'));
+    var button = element(by.className('btn'))
+
+    addItem.sendKeys('new task')
+    button.click()
+
+    var listItems = element.all(by.repeater('item in todoCtrl.storedItems'));
+    expect(listItems.get(0).getText()).toEqual('new task Edit task Delete');
+  });
+
+  it('allows list items to be edited', function(){
+    var addItem = element(by.model('todoCtrl.newItem'));
```

Comment:
Many of these vars can be extracted to the top so you don't have to repeat them in individual tests